### ISSUE TITLE
Support for arrow functions in directives?

### ISSUE THREAD

**Issue by benjamminf** [monkberry/monkberry#16]
Currently calling a method in a directive requires having to re-bind the context, which is a little redundant.

``` twig
<button :onclick={{ this.method.bind(this) }}>Run method</button>
```

It'd be nice if it could support arrow functions, and allow for accessing arguments too (like an event object):

``` twig
<button :onclick={{ e => this.method(e) }}>Run method</button>
```

**Comment by elfet** [comment]
Yes this is cool feature. Already in [TODO](https://github.com/monkberry/monkberry/blob/master/TODO.md).

**Comment by benjamminf** [comment]
Ah my bad, didn't see the TODO list. Looking forward to it :+1:
